frontend/public/components/monitoring/dashboards/table.tsx

```diff
@@ -77,11 +79,18 @@ const Table: React.FC<Props> = ({ panel, pollInterval, queries, namespace }) =>
   const { t } = useTranslation();
 
   const tick = () => {
-    Promise.all(
-      queries.map((q) =>
-        safeFetch(getPrometheusURL({ endpoint: PrometheusEndpoint.QUERY, query: q, namespace })),
-      ),
-    )
+    const allPromises = _.map(queries, (query) =>
+      _.isEmpty(query) || query === ''
+        ? Promise.resolve()
```

Comment:
We might not need to return a Promise.resolve as we have no data. Would it be better to skip the whole calculation if we don't have queries yet?

Reply:
Over Slack on 2/8/23, we decided to leave this as is.